pycbc/strain.py

```diff
@@ -354,8 +381,11 @@ def from_cli(opt, dyn_range_fac=1, precision='single'):
                              pd_taper_window) )
         gate_data(strain, gate_params)
 
-
+    strain.injcutter = injcutter
```

Comment:
Suggestions? ... If I'm honest I don't think a variable name will explain this to a novice user, but hopefully `pycbc_inspiral --help` would. But I'm happy to change this if there is something more suitable.